### DISCUSSION TITLE
Shrinkler & doynaxdepack

### DISCUSSION THREAD

**Discussion by tiztrinity** [BartmanAbyss/vscode-amiga-debug#228] [Q&A]
Hi
I'm rying to use Shrinkler to compress some data  (with -d option) and doynaxdepack to decompress it.
Input (using INCBIN) and output (using AllocMem) buffers are passes and aligned to 16bit, but then I get a crash.
Any idea on how to fix it?

Thanks
TiZ

**Comment by BartmanAbyss** [comment]
Sorry I missed your question. Shrinkler and doynaxdepack are incompatible. Get the doynax packer from [here](https://csdb.dk/release/?id=118678)
